redis/client.go

```diff
@@ -0,0 +1,32 @@
+package redis
+
+import (
+	"os"
+
+	"fmt"
+
+	"github.com/go-redis/redis"
+)
+
+//RedisClient is a *redis.Client that can be set to the same client you've created
+var RedisClient *redis.Client
+
+// NewStandardRedisClient generates a preconfigured redis client according to our spec.
+func NewStandardRedisClient(redisHostEnvironmentVariableName string) (client *redis.Client, err error) {
```

Comment:
To keep this function flexible, could it accept a [`*redis.Options`](https://godoc.org/gopkg.in/redis.v6#Options), override the `Addr` to `$REDIS_HOST:6379`, and return the client? Otherwise this package's client instantiation _has_ to be used.

The thought behind this being: if you're using our spec's package, then the you can expect the infrastructure-side contract to be fulfilled with a `REDIS_HOST` env var.

Also, we use separate Redis databases in some of our applications, and not hardcoding it here would be nice.